bokehjs/test/index.coffee

```diff
@@ -30,3 +30,7 @@ blacklist.push('constructor')
 for own key, val of global.window
   if blacklist.indexOf(key) == -1
     global[key] = val
+
+global.HTMLElement = global.window.HTMLElement
+global.Event = global.window.Event
+global.Image = global.window.Image
```

Comment:
Also, given how things are getting more complicated, I would just run tests in a headless web browser.